pkg/api/output.go

```diff
@@ -10,7 +10,7 @@ import (
 )
 
 // DisplayOutput prints the output based on desired variables
-func DisplayOutput(outputs []*Output, outputFormat string, showNonDeprecated bool) error {
+func DisplayOutput(outputs []*Output, outputFormat string, showNonDeprecated bool, targetVersion string, padChar byte) error {
```

Comment:
I think it would make sense to set `padChar` as a package variable since every caller has the same value hardcoded in. That would cut down on the argument number of both `DisplayOutput()` and `tabOut()`

Reply:
Good idea. I made it a package variable and added an init to the tests in order to use the padChar = '-' to make the tests easier.